src/Microsoft.IdentityModel.Protocols/Configuration/HttpDocumentRetriever.cs

```diff
@@ -40,11 +40,12 @@ namespace Microsoft.IdentityModel.Protocols
     /// </summary>
     public class HttpDocumentRetriever : IDocumentRetriever
     {
-        private readonly HttpClient _httpClient;
+        private HttpClient _httpClient;
         private bool _requireHttps;
 
-        public HttpDocumentRetriever() : this(new HttpClient())
+        public HttpDocumentRetriever()
         {
+            _requireHttps = true;
```

Comment:
You can set the default when declaring the variable OR use the new syntax and set it on the property. So this line and the same below can be removed.